docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,274 @@
+worked to rapidly turn napari into a mature library in the scientific Python
+ecosystem. We added a [code of
+conduct](https://napari.org/docs/developers/CODE_OF_CONDUCT.html), a [mission
+and values
+document](https://napari.org/docs/developers/MISSION_AND_VALUES.html), and
+adopted a [community governance
+model](https://napari.org/docs/developers/GOVERNANCE.html) (based on
+scikit-image's, and since adopted with modifications by zarr). These are
+accessible from our [developer resources
+page](https://napari.org/docs/developers.html), together with a [public
+roadmap](https://napari.org/docs/developers/ROADMAP_0_3.html) explaining
+where the core team will devote effort in the coming months.
+
+We are still humbled by the enthusiasm of the community response to napari and we hope that
+the above documents will continue to encourage potential users to join our
+community. We welcome contributions of all kinds and encourage
+you to get in touch with us if you don't see your most wanted feature in our
+roadmap, or as an issue in our [issue
+tracker](https://github.com/napari/napari/issues).
+
+### Getting in touch
+
+We actively use the [image.sc forum](https://forum.image.sc/tag/napari) (under the "napari" tag) to
+help with any issues they might have using napari and have discussions about exciting ways to use napari. If you're new to napari and getting started this is the first place you should go for help.
+
+If you've found a napari bug or have a specific feature request we'd love to hear from you in our [GitHub issues](https://github.com/napari/napari/issues).
+
+
+If you have questions about contributing to napari you can get in touch with the developer team in our [Zulip chat room](https://napari.zulipchat.com). We try and use the chat room mainly for scheduling and coordination and prefer to keep all user help on [image.sc](https://forum.image.sc/tag/napari) and development discussion on [GitHub](https://github.com/napari/napari) to make it easier for people coming new to the project or participating asynchronously to discover and contribute to those conversations.
+
+### IO plugins
+
+Contributors can now easily extend napari to open and save in a variety of file formats,
+both local and remote, through our plugin architecture. The same file formats as before are available
+to read (TIFF, most image file formats supported by imageio, and zarr).
+However, we can now *write* to all these formats, read and write point
+annotations in .csv format. Additionally, we have made it possible for anyone to create
+packages for napari to read and write in any other formats through plugins. You can read about
+our plugin architecture [here](https://napari.org/docs/plugins/index.html).
+
+Want to drag and drop your favorite file format into napari and have it load automatically? See [this
+guide](https://napari.org/docs/plugins/for_plugin_developers.html) to
+understand how to write your own plugin, and see Jackson Maxfield's
```

Comment:
Psst. Can this be changed? My name is actually Jackson Brown :upside_down_face:

Reply:
Only if you change your github handle, too confusing otherwise. 😜

Sorry, will update, thanks! 😅😊😊😊

Reply:
Absolutely no worries. It's not my fault my parents made it impossible to have the name [JacksonBrown on the internet](https://www.google.com/search?hl=en&q=jackson%20brown) :laughing:

Reply:
:-) updated!!